GeoHealthCheck/config_main.py

```diff
@@ -47,6 +47,7 @@
 GHC_NOTIFICATIONS_EMAIL = ['you2@example.com']
 GHC_SITE_TITLE = 'GeoHealthCheck Demonstration'
 GHC_SITE_URL = 'http://host'
+GHC_METADATA_CACHE_SECS = 7200
```

Comment:
Only problem is that `APP.config` is not available in `probe.py`. At multiple places I see now:
 `from GeoHealthCheck.init import APP`. This may cause issues IMO, maybe the strange errors I sometimes see, as each time `APP` is initialized...At least the APP init should be done once, at then accessed as a Singleton...like `APP.get_config()`.